packages/shared/src/hooks/usePlusSubscription.ts

```diff
@@ -0,0 +1,22 @@
+import { useMemo } from 'react';
+import { useAuthContext } from '../contexts/AuthContext';
+import { feature } from '../lib/featureManagement';
+import { useFeature } from '../components/GrowthBookProvider';
+
+export const usePlusSubscription = (): {
+  showPlusSubscription: boolean;
+  isPlus: boolean;
+} => {
+  const { user } = useAuthContext();
+  const isPlus = user?.isPlus || false;
+
+  const plusSubscriptionFeature = useFeature(feature.plusSubscription);
+
+  return useMemo(
+    () => ({
+      showPlusSubscription: isPlus ? true : plusSubscriptionFeature,
```

Comment:
I think we can just only memo this right, not the whole return object.
Else user change might cause retriggers?